source/Ubuntu-R.ufo/fontinfo.plist

```diff
@@ -267,9 +267,9 @@ The scope of the Ubuntu Font Family includes all the languages used by the vario
 		<key>unitsPerEm</key>
 		<integer>1000</integer>
 		<key>versionMajor</key>
-		<integer>0</integer>
+		<integer>1</integer>
 		<key>versionMinor</key>
-		<integer>830</integer>
+		<integer>0</integer>
```

Comment:
This version changing should be in a separate commit.  It should probably match consistently and be `000`

Reply:
Not sure how to do that since these are really just ints internally. Glyphs displays it as "000" though.